go/interpreter/lfvm/instructions_test.go

```diff
@@ -1563,6 +1563,38 @@ func TestGeneralCall_ResultIsWrittenToStack(t *testing.T) {
 	}
 }
 
+func TestGenericCall_HandlesBigProvidedGasValues(t *testing.T) {
+	zero := *uint256.NewInt(0)
+	gas := tosca.Gas(50_000) // value big enough to cover all gas costs
+	tests := map[string]uint256.Int{
+		"maxInt64-1": *uint256.NewInt(math.MaxInt64 - 1),
+		"maxInt64":   *uint256.NewInt(math.MaxInt64),
+		"maxInt64+1": *uint256.NewInt(math.MaxInt64 + 1),
+		"maxUint64":  *uint256.NewInt(math.MaxUint64),
```

Comment:
The names could be improved by adding whether the nestedGas is updated in the if check or not.

Reply:
I thought this names would be better in case of a failure then you would see the name of the test and the case that fails. Not sure naming based on the effect it should have would help readability that much. I will try and let you know.

Reply:
Since the condition is removed this comment is outdated.